src/utils/chartJs/generateOptions.js

```diff
@@ -11,9 +11,9 @@ const generateOptions = (options = {}) => {
             xAxes: [{
                 type: 'time',
                 time: {
```

Comment:
You can probably pass in options a `time` object and do something in here like this:
```
time: processTimeOption(time)
```
This new function will allows us to affect graphs differently.

You can add some props to PerfherderGraphContainer to affect what options ChartJsWrapper will receive:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/containers/PerfherderGraphContainer/index.jsx#L24
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/containers/PerfherderGraphContainer/index.jsx#L30